### PR TITLE
Product pagination

### PR DESCRIPTION
**Pull request by Trishana9**
- [x] Product listing
 - [x] Pagination
 - [ ] Product detail
 - [ ] Slider in product detail

### CODE REVIEW

File: src/components/ProductListing.vue [hunk 1]
```diff
@@ -1,22 +1,79 @@
...
+  methods:{
+    calculateDiscoutedAmount(product){
+          return (product.price-((product.price*product.discountPercentage)/100)).toFixed(2)
+      },
+
+    displaylist(){
+                let start = this.perPage * this.currentPage;
+                let end = start + this.perPage;
+                this.paginatedItems =  this.productList.slice(start, end);
+                console.log(this.paginatedItems)
```
Comment (lamasagar) [inline src/components/ProductListing.vue:55]: remove console

File: src/components/ProductListing.vue [hunk 2]
```diff
@@ -1,22 +1,79 @@
...
+                        <div class="px-10">
+                            <p>{{product.title}}</p>
+                            <p class="font-bold text-emerald-500">Rs.{{calculateDiscoutedAmount(product)}}</p>
+                            <div class="flex">
+                                <p class="text-gray-400 mr-3 line-through text-sm">{{product.price}}</p>
+                                <p class="text-sm">-{{product.discountPercentage}}%</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
```
Comment (lamasagar) [inline src/components/ProductListing.vue:20]: this could be extracted to a Product.vue component

**Review by lamasagar — CHANGES_REQUESTED** [review]

File: src/components/ProductListing.vue (hunk 1 above)
Comment (Trishana9) [reply src/components/ProductListing.vue:55]: done

File: src/components/ProductListing.vue (hunk 2 above)
Comment (Trishana9) [reply src/components/ProductListing.vue:20]: done

**Pull request closed without merging**